api/localizer/writer.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2019 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+// See proposals/localize-command
+package localizer
+
+import (
+	"net/url"
+	"path/filepath"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	"sigs.k8s.io/kustomize/api/loader"
+	"sigs.k8s.io/kustomize/kyaml/filesys"
+)
+
+const urlSeparator = "/"
+const localizeDir = "localized-files"
+const alpha = "alpha"
+
+var errNewDirRef = errors.New("target references newDir")
```

Comment:
nit: Let's not use the terms `target` and `newDir` as standalone terminology; most users will have no idea what this means unless they've read the documentation and memorized what we decided to call the parameters. Suggest something along the lines of `directory being localized ('target') contains references to localized copy ('newDir')". I'm not sure that's very clear either but maybe you can think of something better

Reply:
Good point

Reply:
I replaced all mentions of `newDir` with `destination` or `localize destination`. 

I forgot this comment included `target` as well. I kept `target` because it's more self-explanatory and I think it's used in existing messages. I also tried to include the `target` value next to mentions of `target`. This being said, I'm still down to rename `target`s to `directory being localized`, `localize directory target`, etc.